src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterPeople = century
+    ? people.filter(person =>
+      century === Math.ceil(person.died / 100) && (person.sex === 'm'))
+    : people.filter(person =>
+      person.sex === 'm');
```

Comment:
you can make this check in filter callback
```suggestion
  const filterPeople = people.filter(person => century
    ? century === Math.ceil(person.died / 100) && (person.sex === 'm'))
    : person.sex === 'm');
```